src/app/core/components/welcome-dialog/welcome-dialog.component.ts

```diff
@@ -17,15 +18,21 @@ export class WelcomeDialogComponent implements OnInit {
   constructor(
     private dialogRef: DialogRef,
     private account: AccountService,
-    private constants: PrConstantsService
+    private constants: PrConstantsService,
+    private api: ApiService
   ) {}
 
-  ngOnInit(): void {
+  async ngOnInit(): Promise<void> {
+    try {
+      const res = await this.api.auth.getInviteToken();
+      this.isEarlyBird = res.token === 'earlyb1rd';
+    } catch (error) {
+      console.log(error);
```

Comment:
```suggestion
```
Can you get rid of this `console.log`? Though that begs the question of what *should* be in that catch block if the API call fails. I do think we should still catch the error (I'm pretty sure throwing it to Sentry will cause the rest of the component to not work properly).

I think in the case of an API failure, we might want to fail silently and set `isEarlyBird` to some default value, since showing the user an error right when they log in for the first time would probably be jarring. But I think it might help if @k8lyn6 weigh in on what should happen if we can't get the user's invite code for whatever reason.

Reply:
If I'm following correctly, I think what you suggested makes sense @meisekimiu . They should probably just get the default (earlybird) message if for some reason we can't get the invite code.

Reply:
For this ticket, @crisnicandrei will just set `isEarlyBird` to `false`. We'll discuss the 1GB separately.